app/code/Magento/Backend/view/adminhtml/ui_component/design_config_form.xml

```diff
@@ -69,7 +69,7 @@
                         <dataScope>search</dataScope>
                     </settings>
                 </field>
-                <field name="theme" formElement="select">
+                <field name="theme" formElement="select" sortOrder="30">
```

Comment:
@JRhyne are you referring to 
```
<actionDelete template="Magento_Backend/dynamic-rows/cells/action-delete" sortOrder="50">
```
? Can we simply remove its sortOrder to achieve the same?

Reply:
Yes, that solution will work as well.  I added sortOrder to the other fields to support adding columns in specific order to that table.  

Should I resubmit the request?

Reply:
Please do a commit amend and force push into the same branch.

As `sortOrder` is not mandatory, I'm quite sure we can omit it. Probably `sortOrder` for these fields may be declared in your theme and then merged altogether.

Reply:
Alright, it's pushed.

Sounds good.